waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/server/FederatedHMSHandler.java

```diff
@@ -323,8 +323,7 @@ public void drop_table_with_environment_context(
       String dbname,
       String name,
       boolean deleteData,
-      EnvironmentContext environment_context)
-      throws NoSuchObjectException, MetaException, TException {
+      EnvironmentContext environment_context) throws NoSuchObjectException, MetaException, TException {
```

Comment:
Can you get the exceptions lined up back the way they were? Will reduce the noise in this PR.

Reply:
Done :)